src/components/shared/BasePopup/index.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CSSTransition } from 'react-transition-group';
+import PopupButtons from './components/PopupButtons';
+import styles from './base.popup.module.scss';
+import './styles.scss';
```

Comment:
What is this? We use scss modules for styles

Reply:
Scss module didn't work with react-transition for me